scripts/genproto.sh

```diff
@@ -27,11 +27,14 @@ if [ ! -f $protoc ] ; then
 fi
 
 echo "using protoc from $protoc"
+if [[ $protoc == "protoc" ]]; then
+	devtools_opts="-I./devtools/include/"
```

Comment:
This directory does not exist, and will never be created on FreeBSD, since scripts/install-protobuf.sh exits whenever `$kernel == FreeBSD`

Reply:
I had a dirty workspace (from previous tests), that's why it worked locally. It's fixed by 3ee68e7.